File: 2-semestre/uri-online-judge/c/1009.c
#include <stdio.h>
 
int main() {
    char nomeVendedor;
    double salarioFixo, totVendas, bonus;
    
    scanf("%s", &nomeVendedor);
    scanf("%lf", &salarioFixo);
    scanf("%lf", &totVendas);
    bonus = (salarioFixo) + (totVendas * 15) / 100;
    
    printf("TOTAL = R$ %.2lf\n", bonus);
    
    return 0;
}

File: 2-semestre/uri-online-judge/java/1009.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        String nomeVendedor;
        double salarioFixo, totalVendas, totalSalario;
        
        Scanner entrada = new Scanner(System.in);
        
        nomeVendedor = entrada.nextLine();
        salarioFixo = entrada.nextDouble();
        totalVendas = entrada.nextDouble();
        totalSalario = ((totalVendas * 15) / 100) + salarioFixo;
        
        System.out.printf("TOTAL = R$ %.2f\n", totalSalario);
    }
}
